docs/UserGuide.adoc

```diff
@@ -187,53 +235,77 @@ Examples:
 * `newtasklist`
 * `nl`
 
-=== Mark a learning outcome as completed : `complete INDEX` `[coming in v2.0]`
+=== Redoing the previously undone command : `redo OR r`
 
-Check a learning outcome as completed based on the given task index.
-Format: `complete INDEX`
+Reverses the most recent `undo` command. +
+Format: `redo OR r`
 
-=== Answering and saving an exercise : `answer INDEX ANSWER` `[coming in v2.0]`
+Examples:
 
-Answer an exercise based off the given question number.
-Format: `answer INDEX ANSWER`
+* `delete 1` +
+`undo` (reverses the `delete 1` command) +
+`redo` (reapplies the `delete 1` command)
 
-****
-* Answer an exercise at the specified `INDEX`.
-* The index refers to the question number.
-* The index must be of format `section number`.`question number`, 2.1.1, 3.2.5, 6.1.7
-****
+* `delete 1` +
+`r` +
+The `redo` command fails as there are no `undo` commands executed previously.
 
-[NOTE]
-====
-`ANSWER` should be in the form of listed choices if the exercise is a multiple choice question. For example, in a question with 4 four choices, one can choose choice 1 and 3 as the answer by using `answer 2.1.1 a c`.
-====
+* `delete 1` +
+`clear` +
+`undo` (reverses the `clear` command) +
+`undo` (reverses the `delete 1` command) +
+`r` (reapplies the `delete 1` command) +
+`redo` (reapplies the `clear` command)
 
-=== Change view to a specific week : `view OR v`
+=== Refreshing the content : `refresh OR rf` `[coming in v2.0]`
 
-Change the browser view to display contents identified by the week number.
-Format: `view INDEX`
+Refreshes the program to update its content.
+Format: `refresh`
```

Comment:
`Format: theme OR t` should be in the next line. Check formatting from command `list` or `sort`.